forage-android/src/main/java/com/joinforage/forage/android/network/data/CheckBalanceRepository.kt

```diff
@@ -134,8 +136,14 @@ internal class CheckBalanceRepository(
                 return ForageApiResponse.Failure(listOf(ForageError(500, "unknown_server_error", "Unknown Server Error")))
             }
 
+            var intervalTime: Long = if (attempt < pollingIntervals.size) {
+                pollingIntervals[attempt]
+            } else {
```

Comment:
Same remark as above but to this part of the code as well
```suggestion
val index = attempt - 1
var intervalTime: Long = if (index < pollingIntervals.size) {
    pollingIntervals[index]
} else {
```